### PR TITLE
update expected files changed in release

### PR DESCRIPTION
**Pull request by spencerdrak**
PR checklist:

- [x] Purpose of the code is [evident to future readers](https://semgrep.dev/docs/contributing/contributing-code/#explaining-code)
- [x] Tests included or PR comment includes a reproducible test plan
- [x] Documentation is up-to-date
- [x] A changelog entry was [added to changelog.d](https://semgrep.dev/docs/contributing/contributing-code/#adding-a-changelog-entry) for any user-facing change
- [x] Change has no security implications (otherwise, ping security team)

If you're unsure on any of this, please see:

- [Contribution guidelines](https://semgrep.dev/docs/contributing/contributing-code)!
- [One of the more specific guides located here](https://semgrep.dev/docs/contributing/contributing/)

### CODE REVIEW

**Review by bmahe — APPROVED** [review]
I am not too familiar with it but it does make sense

File: scripts/release/bump [hunk 1]
```diff
@@ -103,7 +103,7 @@ fi
 
 bump
 
-if git status -s | grep -vq 'CHANGELOG.md\|cli/src/semgrep/__init__.py\|setup.py\|semgrep-core/src/cli/version.ml'; then
+if git status -s | grep -vq 'cli/src/semgrep/__init__.py\|cli/setup.py\|setup.py\|semgrep-core/src/cli-lib/version.ml'; then
```
Comment (zyannes) [inline scripts/release/bump:106]: Do we need to remove `CHANGELOG.md` here?

Comment (spencerdrak) [reply scripts/release/bump:106]: the changelog isn't changed until a later step in the workflow, so I don't think we need to keep this in there anymore.

**Review by zyannes — APPROVED** [review]

File: scripts/release/bump (hunk 1 above)
Comment (spencerdrak) [reply scripts/release/bump:106]: Plus, I think we want to make sure the changelog isn't changing prior to towncrier running.

**Pull request merged**